### DISCUSSION TITLE
Toolbar dropdown open in wrong position

### DISCUSSION THREAD

**Discussion by devopg** [Milkdown/milkdown#674] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/61819835/184549935-8e696939-ca1d-4547-9eec-ac732dbd515d.png)

If place milkdown on div with position fixed - dropdown open in wrong place.

Its bug ?

**Comment by Saul-Mirone** [comment, marked as answer]
You need to make sure the `.milkdown-menu-wrapper` has `position: relative` style.

**Comment by devopg** [comment]
oh with `.milkdown-menu-wrapper` worked ,i try added to `menu-selector-wrapper` but was problem with z-index.
I think better it add mini fix to main code by default.
